### ISSUE TITLE
does it support webpack2?

### ISSUE THREAD

**Issue by luqingxuan** [gowravshekar/font-awesome-webpack#37]
does it support webpack2?

**Comment by KiGniark** [comment]
I just installed it on a project. It works well on webpack2.
You just need to install font-awesome first: 
`npm install font-awesome`

**Comment by AndrewRayCode** [comment]
@KiGniark  no, it does not work with webpack 2: https://github.com/gowravshekar/font-awesome-webpack/issues/33
